lib/elaborate/elaborateNames.ts

```diff
@@ -96,53 +98,56 @@ export class ElaborateNames {
-        this.addObjectsToMap(visibilityMap, ...scopeObj.ports);
+        this.addObjectsToMap(visibilityMap, scopeObj.ports);
       }
       if (directlyVisible && I.implementsIHasGenerics(scopeObj)) {
-        this.addObjectsToMap(visibilityMap, ...scopeObj.generics);
+        this.addObjectsToMap(visibilityMap, scopeObj.generics);
       }
       if (directlyVisible && I.implementsIHasDeclarations(scopeObj)) {
-        this.addObjectsToMap(visibilityMap, ...scopeObj.declarations);
+        this.addObjectsToMap(visibilityMap, scopeObj.declarations);
         for (const type of scopeObj.declarations) {
           if (type instanceof O.OType) {
             if (type instanceof O.OEnum) {
-              this.addObjectsToMap(visibilityMap, ...type.literals);
+              this.addObjectsToMap(visibilityMap, type.literals);
             }
             if (type.units !== undefined) {
-              this.addObjectsToMap(visibilityMap, ...type.units);
+              this.addObjectsToMap(visibilityMap, type.units);
             }
             if (type.protected || type.protectedBody) {
-              this.addObjectsToMap(visibilityMap, ...type.declarations);
+              this.addObjectsToMap(visibilityMap, type.declarations);
             }
             if (type instanceof O.ORecord) {
-              this.addObjectsToMap(recordChildMap, ...type.children);
+              this.addObjectsToMap(recordChildMap, type.children);
             }
           }
         }
       }
       if (directlyVisible && I.implementsIHasStatements(scopeObj)) {
-        this.addObjectsToMap(visibilityMap, ...scopeObj.statements);
+        this.addObjectsToMap(visibilityMap, scopeObj.statements);
       }
       if (directlyVisible && I.implementsIHasLibraries(scopeObj)) {
-        this.addObjectsToMap(visibilityMap, ...scopeObj.libraries);
+        this.addObjectsToMap(visibilityMap, scopeObj.libraries);
       }
     }
   }
 
   getProjectList(searchName: O.OName, libraries: O.OLibrary[]) {
     if (this.projectVisibilityMap === undefined) {
+      // const start = Date.now();
       const projectParser = this.vhdlLinter.projectParser;
       this.projectVisibilityMap = new Map();
-      this.addObjectsToMap(this.projectVisibilityMap, ...projectParser.entities);
-      this.addObjectsToMap(this.projectVisibilityMap, ...projectParser.packages);
-      for (const pkg of projectParser.packages) {
-        this.fillVisibilityMap(pkg);
-      }
-      this.addObjectsToMap(this.projectVisibilityMap, ...projectParser.packageInstantiations);
-      this.addObjectsToMap(this.projectVisibilityMap, ...projectParser.contexts);
-      this.addObjectsToMap(this.projectVisibilityMap, ...projectParser.configurations);
+      this.addObjectsToMap(this.projectVisibilityMap, projectParser.entities);
+      this.addObjectsToMap(this.projectVisibilityMap, projectParser.packages);
+      // What does this do? If I comment out all test succeed but it takes a while
```

Comment:
Might be a left over of where we had a fallback map? Should be fine to remove if all tests succeed...